### PR TITLE
Pause auto materializing by default

### PR DESCRIPTION
**Pull request by johannkm**
And remove the dagit warning

### CODE REVIEW

**Comment by johannkm** [comment]
Current dependencies on/for this PR:
* master
  * **PR #13710** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13710" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/13710?utm_source=stack-comment).

**Comment by johannkm** [comment]
This will also need a docs update

**Review by sryza — APPROVED** [review]
Where does one go in Cloud to turn auto-materialization on/off?

**Comment by johannkm** [comment]
Hmm it’s renamed as agents in cloud… that’s one more reason to find a
better place for this

File: js_modules/dagit/packages/core/src/instance/DeploymentStatusProvider.tsx [hunk 1]
```diff
@@ -35,28 +27,10 @@ export const DeploymentStatusProvider: React.FC<Props> = (props) => {
 
   const codeLocations = useCodeLocationsStatus(!include.has('code-locations'));
   const daemons = useDaemonStatus(!include.has('daemons'));
-  const autoMaterialization = useAutoMaterializationStatus();
```
Comment (salazarm) [inline js_modules/dagit/packages/core/src/instance/DeploymentStatusProvider.tsx:38]: was this removed intentionally?

Comment (johannkm) [reply js_modules/dagit/packages/core/src/instance/DeploymentStatusProvider.tsx:38]: yes- https://elementl-workspace.slack.com/archives/C04UD72HHQX/p1681759584001969?thread_ts=1681506142.934559&cid=C04UD72HHQX

**Pull request merged**